modules/persistent-storage-csi-manila-dynamic-provisioning.adoc

```diff
@@ -25,17 +30,17 @@ To dynamically create a Manila CSI volume using the web console:
 
 . Define the required options on the resulting page.
 
-.. Select the appropriate StorageClass.
+.. Select the appropriate storage class.
 
 .. Enter a unique name for the storage claim.
 
 .. Select the access mode to specify read and write access for the PVC you are creating.
 +
 [IMPORTANT]
 ====
-* Use RWX if you want the persistent volume (PV) that fulfills this PVC to be mounted to multiple Pods on multiple nodes in the cluster.
+* Use RWX if you want the persistent volume (PV) that fulfills this PVC to be mounted to multiple pods on multiple nodes in the cluster.
 
-* Use RWO mode if you want to prevent additional Pods from being dynamically provisioned.
+* Use RWO mode if you want to prevent additional pods from being dynamically provisioned.
```

Comment:
Manila csi driver can not do this. Even the PVC is RWO, the Pods consume this PVC still can be scheduled to multiple nodes.

Reply:
@qinpingli I removed this note about RWO and preventing dynamic provisioning for additional pods. Does this look correct now?